.travis.yml

```diff
@@ -1,26 +1,19 @@
 ---
 language: python
-
-python:
-  - "2.7"
-
-# This (sudo: false) is needed to "run on container-based infrastructure" on
-# which cache: is available
-# http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+python: 2.7
 sudo: false
 
 cache: pip
 
 install:
-  - pip install PyYAML "pandas<0.19" "flake8<3.6.0" yamllint
-  - python setup.py install
+  - pip install -r requirements.txt
```

Comment:
Could this be in the setup.py of idr-utils?

Reply:
At the moment, they are in `idr-utils/requirements.txt` (with `flake8` being uncapped). Can have them in `idr-utils/setup.py` instead. Eventually we should be able to install all requirements as depedency of the utils library.

Reply:
961701b pushed to install the requirements from `idr-utils`